zio-http/src/main/scala/zhttp/http/MediaType.scala

```diff
@@ -0,0 +1,34 @@
+package zhttp.http
+
+import java.util
+
+case class MediaType(
```

Comment:
```suggestion
final class MediaType private[zhttp](
```

The MediaType should only be constructed in `MimeDB`

Reply:
This can be used to add new `MediaType`